src/solver/conv_winoRxS.cpp

```diff
@@ -622,6 +622,10 @@ static bool IsApplicableBase(const ExecutionContext& ctx, const ProblemDescripti
 {
     if(!problem.Is2d())
         return false;
+    if(problem.HasNonPackedTensors())
+    {
+        return false;
+    }
```

Comment:
As far as I know that solver works fine with non-packet tensors. @Kirpich30000 @Slimakanzer, am I correct?

If all the solvers were disabled by-default, it seems we need a ticket to re-enable everything which actually supports strides.

Reply:
Winograd supports arbitrary strides for N, C, H, K, R dimensions in number of elements (fp16 or fp32). W on both input and output, and S on filters should be dense.

Reply:
Non dense W and R is possible, but likely will greatly affect performance. Is there real interest in functional (but not performance) support for this?

Reply:
It still can be faster than naive convolution.

Reply:
adding back the check to WinoGrad solvers at @atamazov 's recommendation